src/bun.js/test/jest.zig

```diff
@@ -1548,6 +1548,16 @@ inline fn createScope(
     var function = if (args.len > 1) args[1] else .zero;
     var options = if (args.len > 2) args[2] else .zero;
 
+    // https://github.com/oven-sh/bun/issues/7849
+    if (!description.isEmptyOrUndefinedOrNull() and description.isCallable(globalThis.vm())) {
```

Comment:
Do you need both the isEmptyOrUndefinedOrNull() in combo with the isCallable() or can it work with just the isCallable?